src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -17,20 +17,24 @@
 <kd-help-section>
   <md-input-container class="md-block" md-is-error="ctrl.isNameError()">
     <label>App name</label>
-    <input ng-model="ctrl.name" name="name" namespace="ctrl.namespace" required ng-pattern="ctrl.namePattern"
-           ng-model-options="{ updateOn: 'default blur', debounce: { 'default': 500, 'blur': 0 } }" kd-unique-name
-           ng-maxlength="ctrl.nameMaxLength">
-    <md-progress-linear class="kd-deploy-form-progress" md-mode="indeterminate" ng-show="ctrl.form.name.$pending">
-    </md-progress-linear>
-    <ng-messages for="ctrl.form.name.$error" role="alert" multiple>
-      <ng-message when="required">Application name is required.</ng-message>
-      <ng-message when="uniqueName">Application with this name
-        already exists within namespace <i>{{ctrl.namespace}}</i>.</ng-message>
-      <ng-message when="pattern">Application name should contain only lowercase letters, numbers, and '-' between words
-      </ng-message>
-      <ng-message when="maxlength">Application name should have no more than 63 characters</ng-message>
-    </ng-messages>
+    <div class="kd-deploy-form-name-container">
+      <input ng-model="ctrl.name" name="name" namespace="ctrl.namespace" required ng-pattern="ctrl.namePattern"
+             ng-model-options="{ updateOn: 'default blur', debounce: { 'default': 500, 'blur': 0 } }" kd-unique-name
+             ng-maxlength="ctrl.nameMaxLength">
+      <div style="height: 2px">
```

Comment:
Can you move this style to a CSS class? Basically, always have all your styles in CSS classes. You can reuse the 2px variable there.